FHEM/00_SIGNALduino.pm

```diff
@@ -663,8 +656,13 @@ SIGNALduino_Set($@)
 		}
 		$hash->{helper}{avrdudecmd} = $flashCommand;
 		$hash->{helper}{avrdudecmd}=~ s/\Q[PORT]\E/$port/g;
-		$hash->{helper}{avrdudecmd} =~ s/\Q[BAUDRATE]\E/$baudrate/g;
 		$hash->{helper}{avrdudecmd} =~ s/\Q[HEXFILE]\E/$hexFile/g;
+		if ($hardware =~ "^nano" && $^O eq 'linux') {
+			$hash->{helper}{avrdudecmd} = $hash->{helper}{avrdudecmd}." || ". $hash->{helper}{avrdudecmd}; 
+			$hash->{helper}{avrdudecmd} =~ s/\Q[BAUDRATE]\E/$baudrate/;
+			$baudrate=115200;
+		}	
+		$hash->{helper}{avrdudecmd} =~ s/\Q[BAUDRATE]\E/$baudrate/;
 		$log .= "command: $hash->{helper}{avrdudecmd}\n\n";
```

Comment:
Ändert es da nicht in beiden avrdudecmd die Baudrate auf 115200?

Reply:
Nein, das passiert meiner Meinung nach nicht